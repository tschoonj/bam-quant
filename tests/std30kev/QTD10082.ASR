File: C:\xrfdaten\2014KW30\chris\std30kev\std10082.spe
Start: 22.07.2014 10:29:55
Stop: 22.07.2014 10:31:35
BESSY: 298.602346 299.018952
IONI: 1.939711E-11 1.943725E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 95860
Run: 100000
Dead: 4
ICR: 7600
OCR: 0
Accumulated BESSY: 59665.8702272
Accumulated Ioni: 3.88343477E-09
N Accumulation: 200
Normfactor:  5.3725016e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pb
exp_x 132.720397949219 132.7204
exp_y 65.4657974243164 65.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  82   2     10.542     91496.       292.      85.22
