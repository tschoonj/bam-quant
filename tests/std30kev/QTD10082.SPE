$DATA:
0 2000
22449
14221
5986
1764
361
48
3
1
0
0
0
0
0
1
0
1
41
196
215
229
219
205
226
212
200
172
171
189
174
170
172
143
157
145
144
142
147
142
134
124
125
138
91
119
98
105
105
98
105
90
96
97
99
94
85
79
80
83
95
106
89
68
73
85
76
76
82
65
68
85
72
80
76
94
84
84
75
68
79
71
80
75
73
75
73
67
73
68
58
62
67
72
62
54
59
61
65
52
86
51
70
77
61
66
68
79
60
59
69
58
45
60
70
71
67
59
57
54
61
50
56
54
64
57
62
50
62
56
54
44
49
44
51
42
44
51
55
57
39
44
43
55
68
63
43
38
46
55
43
50
50
39
57
55
58
49
62
49
44
45
42
41
38
33
33
43
47
51
36
50
43
52
43
41
45
49
38
37
41
42
41
48
39
50
52
49
46
40
41
36
33
38
34
41
49
44
42
56
49
46
36
40
39
51
48
32
35
32
45
40
39
45
52
36
36
42
44
34
42
46
32
35
49
44
44
45
34
40
39
33
34
26
46
47
52
35
33
33
33
31
27
32
45
33
29
41
28
46
37
34
50
38
33
46
38
31
36
37
47
36
41
33
38
36
40
41
39
32
39
43
38
37
34
39
31
48
31
33
39
40
35
44
42
41
43
29
33
28
45
33
31
33
35
24
33
36
37
28
31
38
27
29
29
36
33
39
29
31
33
38
31
43
42
49
55
61
70
74
53
49
59
47
41
37
44
35
24
37
44
43
41
35
27
34
36
29
38
34
44
35
34
37
39
35
36
38
45
49
41
31
44
35
36
30
37
38
36
43
33
34
35
43
36
41
29
34
32
44
49
32
39
48
35
34
37
23
41
39
29
37
41
31
38
30
40
37
45
29
39
48
32
37
34
39
36
37
35
37
35
37
43
33
41
37
39
50
27
48
39
42
35
31
30
40
36
39
38
42
56
39
54
39
35
46
42
43
51
61
28
51
40
40
53
56
53
53
52
50
53
42
63
41
50
44
55
57
42
57
56
75
70
97
95
110
134
133
132
137
133
84
100
70
71
64
62
46
43
55
51
54
39
48
55
55
67
41
54
51
57
50
52
61
51
59
56
52
75
65
68
79
64
58
83
64
72
70
70
79
69
77
71
62
84
91
98
107
131
139
176
214
281
413
672
989
1534
2184
3000
4045
5355
6770
8325
9254
9819
9677
8918
7933
6387
4999
3538
2422
1557
1036
596
403
280
172
170
146
135
134
113
108
112
126
124
101
84
92
106
89
99
92
99
114
157
176
197
253
339
359
419
447
443
431
396
374
325
260
212
203
163
124
129
131
116
150
106
128
111
121
129
132
120
154
131
168
154
154
175
183
174
193
224
265
297
336
407
539
644
738
925
1044
1281
1510
1849
2157
2546
2846
3008
3176
3245
3064
2945
2983
3050
3582
4961
6775
10021
13621
18670
23700
28458
32484
34927
35776
34877
31362
27524
22943
18798
14696
11517
8897
7409
5912
4796
3785
3158
2627
2115
1815
1781
1615
1469
1407
1364
1240
1007
759
696
541
421
343
253
248
235
189
200
230
216
223
194
205
166
174
158
118
95
85
57
65
59
65
57
49
62
38
49
54
46
42
48
45
69
57
70
62
54
53
65
57
54
57
59
70
75
74
72
91
87
107
120
131
161
224
211
229
250
250
247
236
221
196
205
200
192
183
212
297
354
548
768
1164
1660
2129
2935
3559
4428
5100
5449
5793
5477
5020
4509
3744
2923
2318
1740
1334
1052
962
937
1035
1244
1322
1615
1869
2058
2200
2259
2311
2159
2106
1932
1714
1422
1157
899
639
539
390
280
181
160
111
103
90
113
80
106
101
128
187
213
314
309
341
437
447
508
453
478
425
390
317
232
223
143
119
95
50
32
36
25
17
17
17
13
6
6
15
11
18
9
12
6
10
13
10
11
16
13
11
14
13
8
7
14
12
14
8
13
8
15
8
13
7
12
10
9
12
8
10
9
8
16
11
6
10
14
10
14
12
17
10
9
7
4
16
8
13
10
9
12
8
10
8
10
10
15
14
9
11
9
19
15
10
20
25
28
25
31
28
22
30
31
22
23
11
15
24
25
16
14
5
14
11
7
13
7
11
11
16
16
7
12
11
6
13
8
11
9
9
9
10
9
10
6
10
5
10
14
9
13
9
11
9
7
7
7
13
6
10
4
4
12
6
7
8
11
10
10
13
8
10
5
3
9
8
9
5
11
9
6
5
7
10
6
7
8
12
7
5
5
10
11
13
10
12
14
8
11
8
4
8
8
5
6
10
8
7
8
7
8
9
9
9
11
7
5
10
7
3
11
6
7
7
8
10
5
4
10
6
6
9
4
6
7
3
2
7
8
10
5
5
11
0
8
7
2
9
10
8
13
6
5
5
9
4
5
2
9
5
6
4
6
5
5
7
3
5
4
4
4
6
7
9
4
6
4
3
8
5
6
5
6
12
10
9
6
5
9
6
9
4
7
8
5
8
2
7
7
2
10
5
10
3
5
9
6
3
9
5
4
8
3
10
7
7
7
4
9
8
10
8
15
17
18
18
12
9
12
23
14
20
10
17
19
29
12
17
14
17
9
6
9
8
4
13
8
9
3
11
4
5
7
8
6
9
7
10
8
6
5
3
5
2
4
10
5
9
8
10
12
9
11
5
10
7
12
10
13
11
16
12
11
13
9
12
11
15
8
12
9
11
10
5
9
6
9
12
13
11
6
8
11
8
12
8
9
4
8
8
1
2
8
6
11
7
8
5
12
9
11
11
5
7
11
13
6
5
5
10
3
7
6
5
3
13
6
9
5
7
9
2
9
12
8
13
5
9
8
7
8
7
11
11
6
12
13
14
13
15
26
25
28
40
57
85
95
139
189
257
282
302
347
433
464
467
552
560
598
621
687
737
771
775
849
843
831
860
849
726
666
590
535
428
360
259
227
157
96
96
65
28
28
27
17
12
11
11
12
5
6
2
4
7
7
4
5
1
5
1
5
5
5
6
6
6
4
5
6
3
3
8
4
8
3
4
3
11
4
0
5
3
4
6
6
6
8
7
5
4
1
6
5
3
3
3
4
6
6
0
5
5
4
2
7
3
3
4
2
8
3
5
5
5
3
6
5
6
4
9
7
9
5
6
7
8
5
10
8
6
5
9
6
7
13
5
5
4
8
9
3
5
4
14
6
3
9
2
2
6
5
3
6
8
2
4
5
3
4
5
7
5
4
4
7
4
8
7
6
8
9
10
13
22
23
40
46
80
74
82
116
114
148
145
149
155
157
170
181
141
121
112
84
74
57
57
45
25
24
17
9
11
7
8
6
6
9
16
15
12
11
20
25
26
31
26
34
37
26
29
22
24
13
23
22
11
12
10
7
5
8
4
5
6
2
5
3
1
4
5
1
4
4
2
4
3
3
2
2
2
3
3
1
7
3
4
2
4
5
3
3
2
4
2
3
3
3
1
3
5
1
6
3
3
2
3
3
3
6
3
2
5
8
3
5
8
3
3
8
0
3
5
1
4
7
4
6
5
3
4
5
4
2
6
2
2
6
0
2
7
5
5
0
5
5
3
5
3
2
5
4
7
5
6
4
5
4
3
2
5
2
5
3
3
2
5
6
4
9
8
8
5
2
6
3
4
5
2
5
5
9
6
5
3
5
4
2
6
4
5
7
5
7
3
5
5
6
10
7
5
5
3
2
7
8
6
4
12
8
15
12
3
7
9
7
13
6
7
8
6
9
6
9
7
10
12
4
9
16
7
11
8
8
8
14
6
7
5
10
12
14
9
14
5
11
9
12
5
6
4
8
11
10
7
4
6
9
11
8
6
9
9
7
19
8
10
11
11
9
12
11
8
16
11
11
10
8
8
11
8
13
12
15
12
18
12
8
15
14
11
15
17
15
12
14
16
13
24
16
8
15
17
14
20
22
17
7
15
13
20
15
12
13
14
11
21
17
9
22
15
16
28
23
30
30
34
28
30
43
56
56
61
55
80
71
81
88
76
88
88
106
88
81
96
79
101
78
89
73
69
89
89
86
81
72
78
73
63
71
67
73
57
50
49
44
45
25
31
26
17
16
14
11
9
7
8
5
7
3
1
6
0
2
0
1
1
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
2
1
1
1
0
0
0
0
0
0
1
1
1
1
0
0
0
0
0
0
1
1
0
1
0
2
1
0
1
2
0
0
0
1
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10082.spe
Start: 22.07.2014 10:29:55
Stop: 22.07.2014 10:31:35
BESSY: 298.602346 299.018952
IONI: 1.939711E-11 1.943725E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 95860
Run: 100000
Dead: 4
ICR: 7600
OCR: 0
Accumulated BESSY: 59665.8702272
Accumulated Ioni: 3.88343477E-09
N Accumulation: 200
Normfactor:  5.3725016e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pb
 
exp_x 132.720397949219 132.7204
exp_y 65.4657974243164 65.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  82   2     10.542     91496.       292.      85.22
