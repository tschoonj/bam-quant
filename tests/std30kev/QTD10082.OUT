     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:29:02  บ   
     บ  Spectrum: QTD10082.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        4 iterations done    บ   
     บ  ChiSquare =    33.7   last change =   .00%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -39.9 ๑   100.0   บ     -39.9 ๑     1.2   บ   
     บ  GAIN (eV/ch)  บ     20.222 ๑   2.000   บ    20.222 ๑    .002   บ   
     บ det NOISE (eV) บ      162.0 ๑    40.0   บ     162.0 ๑     3.8   บ   
     บ  FANO factor   บ       .061 ๑    .050   บ      .061 ๑    .003   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Pb-L1                            110245. ๑    437.           บ   
     บ    L1M3       12.793    .37802       41675. ๑    166.    111.1  บ   
     บ              634.592    207.41         285.           9.85E-01  บ   
     บ    L1M2       12.306    .34544       38083. ๑    152.     70.5  บ   
     บ              610.510    205.87         301.           9.83E-01  บ   
     บ    L1N3       15.218    .10996       12123. ๑     50.     16.6  บ   
     บ              754.509    214.95         240.           9.87E-01  บ   
     บ    L1N2       15.101    .09081       10011. ๑     43.     19.4  บ   
     บ              748.723    214.59         244.           9.87E-01  บ   
     บ    L1O3       15.765    .03930        4333. ๑     23.     19.4  บ   
     บ              781.558    216.61         227.           9.88E-01  บ   
     บ    L1M5       13.377    .01893        2087. ๑     18.     20.7  บ   
     บ              663.471    209.25         266.           9.87E-01  บ   
     บ    L1M4       13.275    .01262        1391. ๑     17.    144.1  บ   
     บ              658.427    208.93         269.           9.86E-01  บ   
     บ    L1N5       15.453    .00301         332. ๑     15.     15.4  บ   
     บ              766.130    215.66         234.           9.88E-01  บ   
     บ    L1N4       15.427    .00190         210. ๑     15.     15.9  บ   
     บ              764.844    215.58         235.           9.88E-01  บ   
     บ                                                                 บ   
     บ  2 Pb-L2                            424304. ๑    723.           บ   
     บ    L2M4       12.614    .78003      330972. ๑    565.    110.0  บ   
     บ              625.741    206.85         291.           9.85E-01  บ   
     บ    L2N4       14.764    .16802       71292. ๑    122.     80.6  บ   
     บ              732.059    213.56         227.           9.87E-01  บ   
     บ    L2M1       11.349    .02148        9113. ๑     24.    203.5  บ   
     บ              563.186    202.80         336.           9.84E-01  บ   
     บ    L2N1       14.308    .00561        2381. ๑     16.     17.9  บ   
     บ              709.509    212.15         240.           9.87E-01  บ   
     บ    L2O4       15.178    .02276        9655. ๑     23.     16.5  บ   
     บ              752.531    214.83         242.           9.87E-01  บ   
     บ    L2O1       15.053    .00120         510. ๑     16.     28.3  บ   
     บ              746.350    214.44         245.           9.87E-01  บ   
     บ    L2M3       12.127    .00090         381. ๑     18.    150.8  บ   
     บ              601.658    205.30         308.           9.83E-01  บ   
     บ                                                                 บ   
     บ  3 Pb-L3                            118651. ๑    379.           บ   
     บ    L3M5       10.552    .69240       82154. ๑    263.     85.2  บ   
     บ              523.774    200.20         369.           9.80E-01  บ   
     บ    L3M4       10.450    .07873        9342. ๑     36.     81.8  บ   
     บ              518.730    199.87         373.           9.80E-01  บ   
     บ    L3N5       12.623    .14017       16631. ๑     56.    104.4  บ   
     บ              626.186    206.88         290.           9.85E-01  บ   
     บ    L3N4       12.601    .01566        1858. ๑     18.    110.0  บ   
     บ              625.098    206.81         291.           9.84E-01  บ   
     บ    L3M1        9.185    .04029        4781. ๑     26.    492.9  บ   
     บ              456.176    195.67         431.           9.71E-01  บ   
     บ    L3N1       12.143    .01012        1201. ๑     18.    145.1  บ   
     บ              602.450    205.35         307.           9.83E-01  บ   
     บ    L3O1       12.888    .00211         250. ๑     17.    258.8  บ   
     บ              639.290    207.71         282.           9.85E-01  บ   
     บ    L3O5       13.015    .02051        2433. ๑     18.    428.3  บ   
     บ              645.570    208.12         278.           9.86E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                            1447. ๑     51.           บ   
     บ    L2N4       25.229    .29244         423. ๑     18.    645.9  บ   
     บ             1249.542    243.59         116.           7.96E-01  บ   
     บ    L2M1       23.166    .13752         199. ๑     13.      6.2  บ   
     บ             1147.557    237.97         119.           8.67E-01  บ   
     บ    L2N1       27.378    .11932         173. ๑     12.     11.8  บ   
     บ             1355.842    249.32         103.           7.19E-01  บ   
     บ    L3M5       25.408    .07033         102. ๑     11.    448.0  บ   
     บ             1258.404    244.08         115.           7.90E-01  บ   
     บ    L3N5       24.920    .06389          92. ๑     11.    252.3  บ   
     บ             1234.293    242.76         118.           8.07E-01  บ   
     บ    L3M1       27.782    .05414          78. ๑     10.     10.1  บ   
     บ             1375.779    250.37         100.           7.05E-01  บ   
     บ    L3N1       25.316    .02805          41. ๑     11.    587.1  บ   
     บ             1253.858    243.83         115.           7.93E-01  บ   
     บ               23.346    .01654          24. ๑     11.      4.8  บ   
     บ             1156.419    238.47         129.           8.61E-01  บ   
     บ               21.104    .01617          23. ๑     12.      2.1  บ   
     บ             1045.573    232.22         137.           9.27E-01  บ   
     บ               23.064    .01564          23. ๑     11.      5.5  บ   
     บ             1142.514    237.69         119.           8.71E-01  บ   
     บ               23.963    .01525          22. ๑     11.      2.6  บ   
     บ             1186.969    240.16         124.           8.41E-01  บ   
     บ               22.858    .01502          22. ๑     11.      2.4  บ   
     บ             1132.309    237.12         121.           8.77E-01  บ   
     บ               27.558    .01435          21. ๑     10.     15.3  บ   
     บ             1364.704    249.79         102.           7.13E-01  บ   
     บ               27.070    .01303          19. ๑     10.      1.6  บ   
     บ             1340.593    248.50         105.           7.30E-01  บ   
     บ               25.719    .01273          18. ๑     11.     20.6  บ   
     บ             1273.795    244.91         113.           7.79E-01  บ   
     บ               29.528    .01217          18. ๑      9.      3.2  บ   
     บ             1462.142    254.91          89.           6.44E-01  บ   
     บ               29.931    .01105          16. ๑      9.      2.7  บ   
     บ             1482.079    255.94          86.           6.31E-01  บ   
     บ               21.799    .00800          12. ๑     11.      1.8  บ   
     บ             1079.959    234.17         130.           9.09E-01  บ   
     บ               25.100    .00768          11. ๑     11.    565.6  บ   
     บ             1243.155    243.24         117.           8.01E-01  บ   
     บ               28.379    .00727          11. ๑     10.     90.2  บ   
     บ             1405.342    251.94          97.           6.84E-01  บ   
     บ               27.961    .00651           9. ๑     10.      6.5  บ   
     บ             1384.641    250.84          99.           6.98E-01  บ   
     บ               27.473    .00591           9. ๑     10.     12.9  บ   
     บ             1360.530    249.56         102.           7.16E-01  บ   
     บ               25.587    .00423           6. ๑     11.    125.8  บ   
     บ             1267.265    244.56         113.           7.83E-01  บ   
     บ               25.629    .00407           6. ๑     11.     75.1  บ   
     บ             1269.354    244.67         113.           7.82E-01  บ   
     บ               26.922    .00398           6. ๑     10.      2.7  บ   
     บ             1333.293    248.11         106.           7.35E-01  บ   
     บ               21.002    .00368           5. ๑     12.      2.6  บ   
     บ             1040.529    231.93         138.           9.29E-01  บ   
     บ               21.901    .00359           5. ๑     11.      2.5  บ   
     บ             1084.985    234.46         129.           9.06E-01  บ   
     บ               25.991    .00350           5. ๑     11.      3.6  บ   
     บ             1287.254    245.64         111.           7.69E-01  บ   
     บ               24.612    .00349           5. ๑     11.      6.8  บ   
     บ             1219.045    241.93         120.           8.18E-01  บ   
     บ               25.214    .00319           5. ๑     11.    645.9  บ   
     บ             1248.814    243.55         116.           7.97E-01  บ   
     บ               26.113    .00311           5. ๑     11.      3.2  บ   
     บ             1293.270    245.96         110.           7.65E-01  บ   
     บ               24.754    .00265           4. ๑     11.     65.3  บ   
     บ             1226.042    242.31         119.           8.13E-01  บ   
     บ               30.334    .00251           4. ๑      9.      1.6  บ   
     บ             1502.016    256.98          83.           6.18E-01  บ   
     บ               25.889    .00233           3. ๑     11.      3.6  บ   
     บ             1282.210    245.37         112.           7.73E-01  บ   
     บ               19.737    .00188           3. ๑     12.       .8  บ   
     บ              977.975    228.32         153.           9.56E-01  บ   
     บ               23.244    .00188           3. ๑     11.      5.5  บ   
     บ             1151.375    238.19         130.           8.65E-01  บ   
     บ               24.143    .00183           3. ๑     11.      1.2  บ   
     บ             1195.831    240.65         123.           8.35E-01  บ   
     บ               26.317    .00171           2. ๑     10.      2.2  บ   
     บ             1303.357    246.51         109.           7.57E-01  บ   
     บ               22.756    .00171           2. ๑     11.      1.8  บ   
     บ             1127.265    236.84         122.           8.80E-01  บ   
     บ               23.655    .00167           2. ๑     11.      2.7  บ   
     บ             1171.721    239.32         127.           8.51E-01  บ   
     บ               23.949    .00163           2. ๑     11.      2.6  บ   
     บ             1186.259    240.12         124.           8.41E-01  บ   
     บ               30.529    .00148           2. ๑      9.      1.8  บ   
     บ             1511.642    257.47          81.           6.12E-01  บ   
     บ               25.617    .00145           2. ๑     11.     99.2  บ   
     บ             1268.751    244.64         113.           7.82E-01  บ   
     บ               26.516    .00141           2. ๑     10.      2.8  บ   
     บ             1313.206    247.04         108.           7.50E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
